lib/index.js

```diff
@@ -33,6 +33,24 @@ function Viewer(host) {
 
 	socket.on('connect', () => this.connectionState = true );
 
+	socket.on('reassign', data => {
+
+		console.log("Reassign", data);
```

Comment:
This was from the original implementation. I assume there was a reason it was there. Also we already do log lots in production. Seems harmless to me.